mods/ts/weapons/energyweapons.yaml

```diff
@@ -124,7 +124,6 @@ Proton:
 		ImpactSound: ssplash3.aud
 		ValidImpactTypes: Water
 
-# Obelisk laser
```

Comment:
To be fair we don't have any comments on any other weapon across 4 mods, so keeping it is silly. And we make a point of retaining the original names of actors and weapons as they were in the original games (as much as they suck sometimes).